napari/_qt/_tests/test_async_slicing.py

```diff
@@ -10,16 +10,14 @@
 
 # The tests in this module for the new style of async slicing in napari:
 # https://napari.org/dev/naps/4-async-slicing.html
-# They are marked with sync_only because that denotes that the old experimental
-# async should not run as we don't explicitly wait for its threads to finish.
 
 
 @pytest.fixture()
 def rng() -> np.random.Generator:
     return np.random.default_rng(0)
 
 
-@pytest.mark.sync_only
+@pytest.mark.async_only
```

Comment:
I don't think we need any flags for these tests since they directly setup a viewer to do async slicing by modifying `_LayerSlicer._force_sync`. Alternatively, a slightly cleaner way could be to override the setting before creating the viewer since that setting is publicly available now.

We don't need to do the separation at the level of a pytest or CI run, so I think the flags lose their value.